src/extension.ts

```diff
@@ -34,14 +35,19 @@ export const setContextShowLoginMessage = async (value: boolean) => {
 };
 
```

Comment:
this is the main file make sure to add tests here

Reply:
Most of the flows on this file are already tested in our integration tests
but maybe we should consider adding tests for those messages.